custom/icds_reports/static/js/directives/kpi/kpi.directive.js

```diff
@@ -25,6 +25,11 @@ function KpiController($rootScope, $location, haveAccessToFeatures) {
 
     this.isNumber = window.angular.isNumber;
 
+    // Added to hide Infantometer and Stadiometer cards in the UI. To be removed post testing
+    this.toShowInKpi = function (cellLabel) {
+        return !cellLabel.includes('Infantometer') && !cellLabel.includes('Stadiometer');
```

Comment:
hey why its not including all checks as was applied before `(cell.label !== 'AWCs Reported Infantometer' && cell.label !== 'AWCs Reported Stadiometer' && cell.label !== 'Infantometer' && cell.label !== 'Stadiometer'`

Reply:
@challabeehyv

Reply:
@Rohit25negi We are hiding cards with string match infantomter and stadiomter

Reply:
I meant why this check is not needed anymore `cell.label !== 'AWCs Reported Infantometer'`

Reply:
@Rohit25negi we are checking for any text that have Infantomter and Stadiometer which should cover this case.

Reply:
Aaah OK. Sounds good. Thanks